proposals/lip-0052.md

```diff
@@ -147,6 +147,7 @@ The following constants are used throughout the document:
 | `LENGTH_COLLECTION_ID`              | uint32 | 4                       |
 | `LENGTH_TOKEN_ID`                   | uint32 | 8                       |
 | `MAX_LENGTH_DATA`                   | uint32 | 64                      |
+| `MAX_RESERVED_ERROR_STATUS`         | uint64 | 63                      |
```

Comment:
This constant and `CCM_STATUS_OK` should go under the 'Interoperability constants' section